code/modules/mapping/random_rooms.dm

```diff
@@ -1479,3 +1479,139 @@
 	centerspawner = FALSE
 	template_height = 11
 	template_width = 5
+
+/datum/map_template/random_room/skwh_rdm158
+	name = "Large Abandoned Virology"
+	room_id = "skwh_rdm158_abandonedviro"
+	mappath = "_maps/RandomRooms/10x10/skwh_rdm158_abandonedviro.dmm"
+	centerspawner = FALSE
+	template_height = 10
+	template_width = 10
+	weight = 2
+	stock = 3
+
+/datum/map_template/random_room/skwh_rdm157
+	name = "Slime Storage"
+	room_id = "skwh_rdm157_slimestorage"
+	mappath = "_maps/RandomRooms/10x5/skwh_rdm157_slimestorage.dmm"
+	centerspawner = FALSE
+	template_height = 5
+	template_width = 10
+	weight = 3
+
+/datum/map_template/random_room/skwh_rdm159
+	name = "Monkey Testing Room"
+	room_id = "skwh_rdm159_MONKETESTSUBJECT"
```

Comment:
Contains enough samples to get a single virologist every single disease possible.
Contains GBS, Brainrot and Fungal TB, inaccessible diseases for crew.
Contains unused Experimental Viral Base, which was removed from virology after the Extrapolator was introduced.

Reply:
eh i'll remove this map as now looking back at it, i kind of just don't like this map anymore, and in my opinion kind of isn't that much of a fun room to find so yeah